byway/src/main/webapp/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="routepage.css">
```

Comment:
Why is index.html using routpage.css?

Reply:
routepage.css used to be called style.css, but I renamed it since we're now reserving that name for the shared styles. Honestly I'll just remove the styling for index.html from routepage.css